### ISSUE TITLE
python_scripts support

### ISSUE THREAD

**Issue by fsteccanella** [custom-components/hacs#113]
First of all thanks for your awesome work!

**Describe the solution you'd like**
Could be possible to add support for python_scripts?

**Describe alternatives you've considered**
custom_updater support them.

Thanks!

**Comment by ludeeus** [comment]
I'm not sure that would be viable.
Have a look at https://github.com/custom-components/hacs/issues/77 (Request for AppDaemon support)

Do you have examples that can be added?
I'm not aware of a single repository that only contains this?

**Comment by fsteccanella** [comment]
This is the only one that i know:
https://github.com/bieniu/home-assistant-config/blob/master/python_scripts/python_scripts.json

but is not a single repo as you have pointed out.
However we could start defining the integration and let the other devs adapt their repo..

**Comment by ludeeus** [comment]
Setting it up (by using the structure of integrations) would not be that hard (3-5h at most).
But until there is broad support for doing that among developers this is not something I will want to pursue.

**Comment by fsteccanella** [comment]
I understand your concern, but how can a feature spread if it's not supported?
HACS would not exist if HASS had not provided custom_components!

However let's see if this issue will raise interest...

**Comment by VDRainer** [comment]
I would also like to see python_scripts.
One from custom_updater:
```
  python_script_urls:
    - https://raw.githubusercontent.com/pnbruckner/homeassistant-config/master/python_scripts.json
```

**Comment by ludeeus** [comment]
That one would not work under HACS.

**Comment by ludeeus** [comment]
Implemented in [0.9.0](https://github.com/custom-components/hacs/releases/0.9.0) 🎉
